packages/wallet-frontend/src/utils/debounce.ts

```diff
@@ -0,0 +1,16 @@
+import { useEffect, useState } from 'react'
+
+export default function useDebounce(value: string, delay: number) {
```

Comment:
A more generic approach. This way we can use any type for `value`, not only `string`.

```suggestion
export default function useDebounce<T>(value: T, delay: number) {
```

Reply:
Good idea